src/templates/offer.html

```diff
@@ -40,13 +40,14 @@ <h1>Your Donation Swap Offer</h1>
 				</tr>
 			</tbody>
 		</table>
-		<p class="hidden" id="confirmed">Thank you for activating your offer.</p>
+		<div class="hidden" id="confirmed">
+		<p>Thank you for activating your offer.</p>
 		<p>We will send you an email when we find a match. You can create another offer <a href="/start/">here</a>.</p>
 		<p>If you have changed your mind please delete your offer.</p>
+		</div>
 		<button class="hidden" id="btnDelete">Delete Your Offer</button>
 		<div id="message">Loading...</div>
 	</div>
 {%FILE=footer.snippet.html%}
-<script src="/static/offer.js"></script>
```

Comment:
this line has to stay, or else the script won't get loaded, and nothing happens when you click anything.